test/e2e/far_e2e_test.go

```diff
@@ -2,32 +2,62 @@ package e2e
 
 import (
 	"context"
+	"fmt"
 	"time"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	configv1 "github.com/openshift/api/config/v1"
 
+	"github.com/medik8s/fence-agents-remediation/api/v1alpha1"
+	farUtils "github.com/medik8s/fence-agents-remediation/pkg/utils"
+	farE2eUtils "github.com/medik8s/fence-agents-remediation/test/e2e/utils"
+
+	corev1 "k8s.io/api/core/v1"
 	apiErrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
-
-	"github.com/medik8s/fence-agents-remediation/api/v1alpha1"
 )
 
 const (
-	fenceAgentDummyName = "echo"
-	testNamespace       = "openshift-operators"
+	fenceAgentDummyName  = "echo"
+	fenceAgentAWS        = "fence_aws"
+	fenceAgentIPMI       = "fence_ipmilan"
+	fenceAgentAction     = "status"
+	nodeIndex            = 0
+	succeesStatusMessage = "ON"
+	containerName        = "manager"
+
+	// eventually parameters
+	timeoutLogs  = 1 * time.Minute
+	pollInterval = 10 * time.Second
 )
 
 var _ = Describe("FAR E2e", func() {
+	var (
+		far                 *v1alpha1.FenceAgentsRemediation
+		fenceAgent          string
+		clusterPlatformType string
```

Comment:
IMO this var is redundant, we don't need to store it or convert to string, since it's already included in `clusterPlatform`